etl/naaccr_etl_sqlserver.sql

```diff
@@ -1334,9 +1330,186 @@ CREATE TABLE naaccr_data_points_temp
 	WHERE record_id        = det.rec_id
 	AND episode_concept_id = 32531; --Treatment Regimen
 
```

Comment:
We need a DELETE statement for the observation to data from previous runs. Example from above: 

```
DELETE FROM procedure_occurrence
WHERE procedure_type_concept_id = 32534;
```

Reply:
Makes sense.